plugins/home/src/homePageComponents/FeaturedDocs/FeaturedDocs.tsx

```diff
@@ -0,0 +1,154 @@
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import useAsync from 'react-use/lib/useAsync';
+import {
+  LinkButton,
+  EmptyState,
+  Link,
+  CodeSnippet,
+  InfoCard,
+  Progress,
+  WarningPanel,
+} from '@backstage/core-components';
+import { catalogApiRef, CatalogApi } from '@backstage/plugin-catalog-react';
+import { useApi } from '@backstage/core-plugin-api';
+import { EntityFilterQuery } from '@backstage/catalog-client';
+import { ClassNameMap } from '@material-ui/styles';
+
+import { makeStyles, Theme, Typography } from '@material-ui/core';
+
+type DocsCardProps = {
+  filter: EntityFilterQuery;
+  color?: 'inherit' | 'primary' | 'secondary' | undefined;
+  customStyles?: ClassNameMap<string> | undefined;
+  emptyState?: React.ReactNode | undefined;
+  subLinkText?: string | undefined;
+  title?: React.ReactNode | string | undefined;
+};
+
+const useStyles = makeStyles<Theme>(() => ({
+  docDescription: {
+    fontSize: 16,
+    fontWeight: 400,
+    marginBottom: '16px',
+    marginTop: '12px',
+  },
+  docSubLink: {
+    fontSize: 12,
+    fontWeight: 700,
+    lineHeight: 2,
+  },
+  docsTitleLink: {
+    fontSize: 16,
+    fontWeight: 600,
+    lineHeight: 2,
+  },
+}));
+
+/**
+ * A component to display specific Featured Docs.
+ * @param {EntityFilterQuery} filter - The entity filter used to display only the intended item/s
+ * @param {'inherit' | 'primary' | 'secondary' | undefined} [color] - An optional color which can be customized through themes
+ * @param {ClassNameMap<string> | undefined} [customStyles] - An optional ClassNameMap created with makeStyles
+ * @param {React.ReactNode | undefined} [emptyState] - An optional ReactNode for empty states
+ * @param {string | undefined} [subLinkText] - An optional string to customize sublink text
+ * @param {React.ReactNode | string | undefined} [title] - An optional string or ReactNode to customize the card title
+ *
+ * @public
+ */
+export const FeaturedDocs = (props: DocsCardProps) => {
+  const { color, customStyles, emptyState, filter, subLinkText, title } = props;
+  const linkText = subLinkText || 'LEARN MORE';
+  const defaultStyles = useStyles();
+  const styles = customStyles || defaultStyles;
+  const catalogApi: CatalogApi = useApi(catalogApiRef);
+  const {
+    value: entities,
+    loading,
+    error,
+  } = useAsync(async () => {
+    const response = await catalogApi.getEntities({
+      filter: filter,
+    });
+    return response.items;
+  });
+
+  if (loading) {
+    return <Progress />;
```

Comment:
It's often best to render the card always, and then have a separate inner `<Content>` component (private) which does all the loading and showing of progress and errors. In a grid of cards, it's confusing to see a free-floating progress bar or warning panel without context of what's going on